tests/test_hf_api.py

```diff
@@ -109,37 +106,36 @@ class HfApiCommonTest(unittest.TestCase):
 
 class HfApiLoginTest(HfApiCommonTest):
     def setUp(self) -> None:
-        erase_from_credential_store(USER)
+        erase_from_credential_store(USERNAME_PLACEHOLDER)
```

Comment:
Why is this change needed?

Reply:
Because beforehand we were saving the username/password in the credential cache, and that's not the case anymore!

Now we're saving `hf_user:token` instead, with `USERNAME_PLACEHOLDER` being `hf_user`.